stores-api/assets-api/src/main/java/com/bloxbean/cardano/yaci/store/api/assets/controller/AssetController.java

```diff
@@ -2,96 +2,129 @@
 
 import com.bloxbean.cardano.yaci.store.api.assets.service.AssetService;
 import com.bloxbean.cardano.yaci.store.assets.domain.TxAsset;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import jakarta.validation.constraints.Max;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.Pattern;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 import org.springframework.web.server.ResponseStatusException;
 
+import java.math.BigInteger;
 import java.util.List;
 
+@Slf4j
 @RestController
-@RequestMapping("${apiPrefix}")
 @RequiredArgsConstructor
-@Slf4j
+@RequestMapping("${apiPrefix}/assets")
 public class AssetController {
+
     private final AssetService assetService;
 
-    @GetMapping("/txs/{txHash}/assets")
-    public List<TxAsset> getAssetTxsByTx(@PathVariable String txHash) {
+    @Tag(name = "Asset Service")
+    @GetMapping("/txs/{txHash}")
+    @Operation(summary = "Assets Information by Tx Hash", description = "Returns the information for all assets included in a transaction.")
+    public List<TxAsset> getAssetTxsByTx(@PathVariable @Pattern(regexp = "^[0-9a-fA-F]$") String txHash) {
         return assetService.getAssetsByTx(txHash);
     }
 
-    @GetMapping("/assets/txs/fingerprint/{fingerprint}")
-    public List<TxAsset> getAssetTxsByFingerprint(@PathVariable String fingerprint, @RequestParam(name = "page", defaultValue = "0") int page,
-                                                @RequestParam(name = "count", defaultValue = "10") int count) {
+    @Tag(name = "Asset Service")
+    @GetMapping("/fingerprint/{fingerprint}")
+    @Operation(summary = "Asset Information by Fingerprint", description = "Returns asset information by fingerprint.")
```

Comment:
This endpoint provides asset's minting/burning transaction information.

Reply:
So should I call it Asset History like in Koios ?
https://api.koios.rest/#get-/asset_history